src/lib/public/confetti/new-shapes.ts

```diff
@@ -0,0 +1 @@
+export default 'url(/assets/1st/basic_shapes.svg)';
```

Comment:
Nit: In acmcsuf.com, we tend to use [kabob case](https://en.wikipedia.org/wiki/Kebab_case) for storing our files.
```suggestion
export default 'url(/assets/1st/basic-shapes.svg)';
```